_includes/markdown/Markup.md

```diff
@@ -285,10 +285,96 @@ Combining automated and manual testing practices allows 10up to maintain a high
+At 10up, we employ progressive enhancement to ensure that the sites we build for our clients are accessible to as many users as possible.
 
 ### Polyfills
 When writing markup that does not have wide browser support, using polyfills can help bring that functionality to those older browsers. Providing support for older browsers is incredibly important to the business objectives of our clients. In an effort to prevent code bloat, we only provide polyfills for features that are functionally critical to a site.
 
 ### Feature Detection
 At 10up, the concept of feature detection is used to test browser support for new features that do not yet have full support across the board. The concept of feature detection is to test if a feature is supported by the browser and if not supported, conditionality run code to provide a similar experience with browsers that do support the feature. While popular [feature detection libraries](https://modernizr.com/) exist, there are [feature detection techniques](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection#JavaScript) for JavaScript and [@supports](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) at-rule for CSS that can be utilized.
+
+<h2 id="svg" class="anchor-heading">SVG {% include Util/top %}</h2>
+<abbr title="Scaleable Vector Graphics">SVG</abbr> has become a prevalent means for displaying rich vector graphics. At 10up we prefer to use SVGs over icon fonts for several reasons:
+
+1. Sharper in fidelity - icon fonts can be a bit pixelated, because the browser treats them as text (i.e. anti-aliased). While this can sometimes be overcome by adjusting `font-smoothing`. This approach is not recommended.
+2. Broader CSS control - individual stroke and fill properties can be adjusted.
+3. Semantics - An `<svg>` element has semantic meaning as an image. This helps communicate to browsers and assitive technology the intended meaning of the element.
+4. Accessibility - while font icons _can_ be implemented in an accessible way. There are obscure instances where glyphs have mixed Unicode meaning and can be output in inconsistent manner. Whereas, an SVG is output _as is_.
+
+### Best Practices &amp; Common Usage
+
+#### SVG icon in a button with text
+
+```
+<button type="button">
+    Menu
+    <svg viewBox="0 0 10 10"
+        class="svg-icon"
+        aria-hidden="true"
+        focusable="false">
+        <path d="m1 7h8v2h-8zm0-3h8v2h-8zm0-3h8v2h-8z"/>
+    </svg>
+</button>
+```
+
+Since the SVG is merely decorative and the "Menu" textually explains the button's purpose. Then we can leverage `aria-hidden="true"` and `focusable="false"` to hide the SVG-as-decorative-element from the accessibility tree.
```

Comment:
Can you add more details around using focusable="false", specifically why it's needed (and why tabindex="-1" wouldn't suffice). I don't think people are generally aware of the SVG2.0 spec support in IE